CHANGELOG-1.6.md

```diff
@@ -20,6 +20,9 @@ and date `## vX.Y.Z - YYYY-MM-DD` and create a new placeholder section for  `unr
 
 * [CHANGE] Upgrade Stargate to v1.0.63
 * [CHANGE] Upgrade cass-operator to v1.10.4
+* [CHANGE] Deprecate all K8ssandra v1.x charts, refer users to k8ssandra-operator.
+* [CHANGE] Upgrade Stargate to v1.0.61
+* [CHANGE] Upgrade cass-operator to v1.12.0
```

Comment:
k8ssandra 1.x does not use 1.12.0 (it should use 1.10.5), this shouldn't be part of the changelog.

Reply:
That was part of the rebase I did. Maybe I shouldn't be branching off master and should branch off a release branch? Some of our projects still use tags on main and some use release branches, what's our strategy on this one?

Reply:
>That was part of the rebase I did

I just looked at `charts/k8ssandra/Chart.yaml` in the PR and it still depends on version 0.35.3 of the cass-operator chart, which installs cass-operator 1.10.4. Due to API changes in Kubernetes in 1.22 I believe I think we have avoided upgrading the k8ssandra chart to 1.11 or later of cass-operator.

Reply:
OK, tbh I don't know where these changes came from, I'm just going to remove them as you ask. We have another convo going offline about the fact that we're still supporting cass-operator 1.10.x.